### ISSUE TITLE
Login to Bitbucket stop to work

### ISSUE THREAD

**Issue by ivangusev** [leonhartX/gas-github#144]
Please fix, currently it throws "Can not login, please check your username and password."

**Comment by shaniceweiwei** [comment]
same issue

**Comment by leonhartX** [comment]
umm, maybe bitbucket's API has some changes,will take some time to look at this

**Comment by steeleCoding** [comment]
Also Same issue

**Comment by ashraftalib** [comment]
@leonhartX 

 I logged out from SCM. Now I am not able to login from SCM and getting the below message:
"Can not login, please check your username and password." 
![Login issue from SCM to Bitbucket](https://user-images.githubusercontent.com/31342456/117758690-1c74cd80-b240-11eb-9e2f-10ad06d34cde.PNG)

**Comment by leonhartX** [comment]
did some investigation, the bitbucket API `https://bitbucket.org/site/oauth2/access_token` itself is still fine, but it doesn't respond to the `OPTIONS` preflight request correctly and return a 500, so chrome stop sending the real `POST` request

I'm not sure what we can do for that since we can not bypass CORS in chrome

**Comment by leonhartX** [comment]
@ashraftalib this may related to the Apps Script API not enabled 
see #135

**Comment by ashraftalib** [comment]
@leonhartX 
I already enabled Apps Script API but still the same issue I am getting.
![API Enabled already](https://user-images.githubusercontent.com/31342456/117794805-34ae1200-b26b-11eb-9f57-1deb96873845.PNG)

**Comment by ivangusev** [comment]
@leonhartX So it's error on bitbucket server side. Can you try to submit an issue to bitbucket? Your addon is very popular and has no alternative, so i guess they would put more attention to the issue.

**Comment by nattila1** [comment]
> @ashraftalib this may related to the Apps Script API not enabled
> see #135

No, it is not related to API. Event though API is enabled to me, I always get the same error: "TypeError: Cannot read property 'forEach' of undefined"

**Comment by zensarCompass** [comment]
I am also facing this issue. Is someone from the team working on this?

**Comment by leonhartX** [comment]
Hey guys, does the issue still exist?
I check it today and found bitbuckt works again without any CORS issue (maybe they fixed it)

**Comment by gauranggurukul** [comment]
Hi All,

The SCM login for Bitbucket is still not working. Do we have any updates on this issue?

**Comment by cbhouston** [comment]
Still same issue for me, cannot login BitBucket

**Comment by Shaws-WD** [comment]
the problem persists with CORS issue preventing bitbucket login  :-) 
I am trying gas-github for the first time.  I would love to use it but can't login to bitbucket
